src/Classes/ModStore.lua

```diff
@@ -501,8 +501,36 @@ function ModStoreClass:EvalMod(mod, cfg)
 				return
 			end
 		elseif tag.type == "SocketedIn" then
-			if not cfg or tag.slotName ~= cfg.slotName or (tag.keyword and (not cfg or not cfg.skillGem or not calcLib.gemIsType(cfg.skillGem, tag.keyword))) then
+			if not cfg or tag.slotName ~= cfg.slotName or (tag.socketColor ~= nil and (tag.sockets == nil or (tag.sockets ~= nil and type(tag.sockets) ~= "string")) and tag.socketColor ~= cfg.socketColor) or (tag.keyword and (not cfg or not cfg.skillGem or not calcLib.gemIsType(cfg.skillGem, tag.keyword))) then
 				return
+			elseif tag.sockets ~= nil and tag.sockets == "all" and tag.socketColor ~= nil then
```

Comment:
You could probably pull tag.sockets ~= nil out of the if statements to avoid calling it multiple times.